src/frontend/apps/desk/src/features/teams/__tests__/PanelTeams.test.tsx

```diff
@@ -30,14 +30,19 @@ describe('PanelTeams', () => {
     ).toBeInTheDocument();
   });
 
-  it('renders with empty team to display', async () => {
+  it('renders a team empty', async () => {
```

Comment:
```suggestion
  it('renders an empty team', async () => {
```

Reply:
It might be a bit misleading, as the team is not empty. We might want to test an empty team, and a team with only its owner, we should both lead to the same display

Reply:
[216f184](https://github.com/numerique-gouv/people/pull/43/commits/216f1842596371fe67ad85a880c55c71ba66ccb9)

Reply:
wrong article it's `an`

Reply:
[eabcd9b](https://github.com/numerique-gouv/people/pull/43/commits/eabcd9b9f003a3c38618d4758e12da481fee0021)